communication.md

```diff
@@ -0,0 +1,75 @@
+# Communicating in Gizra
+
+Let’s talk about communicating internally in Gizra.  
+
+Today, Gizra’s main office is in Tel Aviv Israel (GMT +2) with 16 Israeli employees that use the office in
+varied ways, ranging anywhere from showing up 5 days/week during traditional work hours (~6 people) all the
+way
+to working remotely and coming into the office once/week (~2 people). There is a lot of flexibility both in
+work times and where employees sit and this is mostly determined by role and the person’s circumstances.  
+
+Gizra’s recent hiring strategy has introduced a wonderful group of distributed employees that form almost half
+the company, span across 7 countries (North America and Europe), and several timezones.
+
+This has been wonderful for Gizra but it also comes with the need to rethink how we communicate with each
+other. This is very important and we ALL need to be mindful of it.  
+
+##Tools we have available
+
+###Github
+This is where the project development is communicated.
+The chapters about [Pull Requests] (https://www.thegizraway.com/pull_requests.html) covers how to communicate in Github. However, it is worth mentioning that the
+goal is to write liberally and make things easy to understand so anyone, at anytime, located anywhere can
+read it, and know how to proceed. Note: communicating with clients occurs in the issues, and not in the PR's.
```

Comment:
Not sure this is the place (maybe there's something more specific elsewhere in terms of client work), but this could use some further explanation (you gave a nice one the other day) - also point out possible exceptions.